realm/realm-library/src/main/java/io/realm/RealmResults.java

```diff
@@ -62,6 +54,18 @@
  * @see Realm#executeTransaction(Realm.Transaction)
  */
 public class RealmResults<E extends RealmModel> extends OrderedRealmCollectionImpl<E> {
+    public static <T extends RealmModel> RealmResults<T> getBacklinkResults(BaseRealm realm, Row row, Class<T> srcTableType, String srcFieldName) {
+        if (!(row instanceof UncheckedRow)) {
+            throw new IllegalArgumentException("Row is " + row.getClass());
+        }
+        UncheckedRow uncheckedRow = (UncheckedRow) row;
+        Table srcTable = realm.getSchema().getTable(srcTableType);
```

Comment:
Yeh.  Great question.  I have no idea.  Will be testing DynamicRealm stuff over the next few days.